Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/resource/quantity.go

```diff
@@ -88,34 +86,19 @@ import (
 // cause implementors to also use a fixed point implementation.
 //
 // +protobuf=true
-// +protobuf.embed=string
+// +protobuf.embed=QuantityProto
 // +protobuf.options.marshal=false
 // +protobuf.options.(gogoproto.goproto_stringer)=false
 type Quantity struct {
-	// i is the quantity in int64 scaled form, if d.Dec == nil
-	i int64Amount
-	// d is the quantity in inf.Dec form if d.Dec != nil
-	d infDecAmount
-	// s is the generated value of this quantity to avoid recalculation
-	s []byte
+	// Amount is public, so you can manipulate it if the accessor
```

Comment:
ruh!?!? was this a reversion upstream? see https://github.com/appc/spec/pull/607/files#diff-6c84e496be2b46a45cb5c0b30224f6ebL17